applications/nrf5340_audio/tools/buildprog/nrf5340_audio_dk_devices.py

```diff
@@ -74,11 +74,11 @@ class Location(Enum):
     FRONT_RIGHT_WIDE = (1 << 25, "Front Right Wide")
     LEFT_SURROUND = (1 << 26, "Left Surround")
     RIGHT_SURROUND = (1 << 27, "Right Surround")
+    NA = (1 << 28, "Not Applicable")
```

Comment:
Probably can't do this. This could potentially end up in the location field over BLE where this is not a defined value.